cluster/manifests/zmon-agent/deployment.yaml

```diff
@@ -4,17 +4,17 @@ metadata:
   name: "zmon-agent"
   namespace: "visibility"
   labels:
-    application: "zmon-agent"
+    application: "zmon-agent-core"
     version: "0.1-a56-zv2"
 spec:
   replicas: {{.ConfigItems.zmon_agent_replicas}}
   selector:
     matchLabels:
-      application: zmon-agent
+      application: zmon-agent-core
```

Comment:
This won't work. We need to delete the deployment first and re-apply.

Reply:
BTW, that would be totally fine if you "move" the [zmon-agent](https://yourturn.stups.zalan.do/application/detail/zmon-agent) app from our account to yours — if this is possible — thus, no re-deployment will be needed. Anyway, we use [zmon-agent-aisasfoverarch](https://yourturn.stups.zalan.do/application/detail/zmon-aisasfoverarch) for our needs.

Reply:
@nolar thanks, then I propose we do that instead. Is that ok with your @sbrauner-z ?

Reply:
@mikkeloscar @nolar sounds good to me if this is actually possible.

Reply:
Oops. Just noticed it is a public GitHub. Removed the internal links & editing history.